tools-reference/cat/text.xml

```diff
@@ -73,7 +73,7 @@ tells <c>cat</c> to strip leading tabs (please note that when you want
 to copy the example, you have to replace the leading spaces with
 tabs), so that, as Azarah puts it, "we won't have the ebuilds looking
 from space effect". For such small files as the example above, a set
-of here documents is more elegant, and easier to maintain, than the
+of here documents (heredocs) is more elegant, and easier to maintain, than the
```

Comment:
I've never heard the term "heredocs" before. The bash manual doesn't mention it either.

Reply:
Wikipedia:
>In computing, a here document (here-document, here-text, heredoc, hereis, here-string or here-script) is a file literal or input stream literal: [...]

If you _really_ want, we can drop it, but I think adding jargon is useful so people learn.

Reply:
Yes, please drop. "Here document" is clear enough, and is the term used in the Bash manual (most of the time, with an hyphen, so arguably we could do that as well).

Reply:
Sure.